appendix-basic65.tex

```diff
@@ -2100,9 +2100,9 @@ \subsection{COPY}
 
 \item [Usage:]    Copies a file to another file, or one or more files from one disk to another.
 
-                  {\bf source} is either a quoted string, e.g. \screentext{"DATA"} or a string expression in brackets, e.g. \screentext{(FI\$)}.
+                  {\bf source} is either a quoted string, e.g. \screentext{"DATA"} or a string expression in brackets, e.g. \screentext{(FI\$)}
```

Comment:
If this is an argument description, it can be a sentence fragment without sentence ending punctuation if you remove "is," otherwise we need an ending period and/or a rewrite to avoid putting sentence punctuation next to typing text. (I'm fine with the definitional sentence fragment ending with a period, in general. Not a big deal if we'll do a pass over argument lists later.)